packages/swirl-components/src/components/swirl-tree-view-item/swirl-tree-view-item.tsx

```diff
@@ -27,6 +27,7 @@ export class SwirlTreeViewItem {
   @Element() el!: HTMLSwirlTreeViewItemElement;
 
   @Prop() active?: boolean;
+  @Prop() expandable?: boolean = true;
```

Comment:
Suggestion: If `false` the component shouldn't just hide the toggle icon, but also not be expandable via `expand()`.

Reply:
disabled the methods when expandable is false